bot-components/GitHub_mutations.ml

```diff
@@ -141,30 +150,17 @@ let update_check_run ~bot_info ~check_run_id ~repo_id ~conclusion ?details_url
 
 (* TODO: use GraphQL API *)
```

Comment:
This comment will need to move below since the two functions below were converted to use GraphQL.